CommonTools/PileupAlgos/python/PhotonPuppi_cff.py

```diff
@@ -6,10 +6,11 @@
                              puppiCandName  = cms.InputTag('puppi'),
                              photonName     = cms.InputTag('reducedEgamma','reducedGedPhotons'),
 			     recoToPFMap    = cms.InputTag("reducedEgamma","reducedPhotonPfCandMap"),
-                             photonId       = cms.InputTag("egmPhotonIDs:cutBasedPhotonID-Spring16-V2p2-loose"), 
-                             pt             = cms.double(10),
+                             photonId       = cms.InputTag(""), # Add once available and validated (Note that Spring16 cut based photon ID fails for high pT photons in 93X)
```

Comment:
IIUC, the comment about Spring16 ID is not particularly relevant and it better be removed.
(also, it refers to the previous commit state rather than to the future unused photon IDs)